integration-tests/package.json

```diff
@@ -16,7 +18,8 @@
     "start": "cypress open",
     "start:dev": "cypress open --config-file cypress-dev.config.ts",
     "format-lint": "prettier --list-different \"cypress/**/*.ts\"",
-    "eslint": "npx eslint ./cypress/**/*.ts"
+    "eslint": "npx eslint ./cypress/**/*.ts",
```

Comment:
I would rather have a dedicated project for playwright instead of mixing stuff from both Cypress and Playwright at the same place.